### ISSUE TITLE
find_options can't handle `undefined` as last arg

### ISSUE THREAD

**Issue by beamery-tomht** [williamkapke/mongo-mock#133]
I'm working with a wrapper around our connection to mongo. It will always call:

```ts
return await client[queryType](dataSource, query, queryOptions);
```
for a given collection method.

When `queryOptions` is undefined, the `find_options.js` will see a signature of `object,undefined` and throw an error. This can easily be interpreted as a signature of just `object`... 

Can you fix `find_options` to strip the trailing `undefined` please and not throw in these scenarios.

**Comment by cprhokie** [comment]
created a pull request for this issue

**Comment by beamery-tomht** [comment]
Thanks, still waiting for this https://github.com/williamkapke/mongo-mock/pull/141

**Comment by LinusU** [comment]
This is fixed in [🚢 4.1.0 / 2021-08-12](https://github.com/williamkapke/mongo-mock/releases/tag/v4.1.0)
